src/cool_seq_tool/mappers/exon_genomic_coords.py

```diff
@@ -25,7 +25,7 @@ def _check_errors(
     required_fields: list[str],
     either_or_fields: list[tuple[str, str]] | None = None,
 ) -> dict:
-    """Ensure that required fields are set if `errors` field is empty
+    """Ensure that required fields are set if ``errors`` field is empty
```

Comment:
I also end up using a lot of markdownish/RSTish formatting in docstrings regardless, but fwiw private methods don't show up in the docs so it's NBD if they aren't formatted